gateway/modules/schema/update.go

```diff
@@ -15,36 +17,44 @@ func (s *Schema) ValidateUpdateOperation(dbAlias, col, op string, updateDoc, fin
 	}
 	schemaDb, ok := s.SchemaDoc[dbAlias]
 	if !ok {
+		logrus.Errorf("error validating update operation in schema module dbAlias (%s) not found in schemaDoc of schema module", dbAlias)
 		return fmt.Errorf("%s is not present in schema", dbAlias)
 	}
 	SchemaDoc, ok := schemaDb[col]
 	if !ok {
+		logrus.Infof("validating update operation in schema module collection (%s) not found in schemaDoc where dbAlias (%s)", col, dbAlias)
 		return nil
 	}
 
 	for key, doc := range updateDoc {
 		switch key {
+		case "$unset":
```

Comment:
Unset should be allowed only on non required fields for mongo, and JSONB fields for postgres. MySQL and Sqlserver not allowed